From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Subject: kstrtox: consistently use _tolower()
Date: Thu, 17 Aug 2023 17:59:19 +0300

We already use _tolower() in other places, so convert the one which open
codes it.

Link: https://lkml.kernel.org/r/20230817145919.543251-1-andriy.shevchenko@linux.intel.com
Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/kstrtox.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/kstrtox.c~kstrtox-consistently-use-_tolower
+++ a/lib/kstrtox.c
@@ -59,7 +59,7 @@ unsigned int _parse_integer_limit(const
 	rv = 0;
 	while (max_chars--) {
 		unsigned int c = *s;
-		unsigned int lc = c | 0x20; /* don't tolower() this line */
+		unsigned int lc = _tolower(c);
 		unsigned int val;
 
 		if ('0' <= c && c <= '9')
_
